docs/reference/release-notes/7.0.0-alpha2.asciidoc

```diff
@@ -1,10 +1,599 @@
+* [ML-DataFrame] add a client helper origin for data frame {pull}36057[#36057]
+* [ML-DataFrame] Rename plugin and endpoints {pull}36048[#36048]
+* [ML-Dataframe] Feature/fib cleanups {pull}35995[#35995]
+* [ML-DataFrame] add a stats endpoint {pull}35911[#35911]
+* [ML-DataFrame]remove settings from transport actions {pull}35831[#35831]
+* [ML-Dataframe] Feature/fib get jobs {pull}35825[#35825]
+* [ML-Dataframe] re-factor code to use LogManager and own loggers {pull}35631[#35631]
+* [ML-Dataframe] change licensing check and plugin name/description {pull}35618[#35618]
+* Add delayed datacheck to the datafeed job runner {pull}35387[#35387] (issue: {issue}35131[#35131])
+* [ML-Dataframe] add validation and mapping detection {pull}34844[#34844]
+* [ML-Dataframe] add integration tests {pull}34554[#34554]
+
+Mapping::
+* Make typeless APIs usable with indices whose type name is different from `_doc` {pull}35790[#35790] (issue: {issue}35190[#35190])
 
+SQL::
+* Introduce HISTOGRAM grouping function {pull}36510[#36510] (issue: {issue}36509[#36509])
+* DATABASE() and USER() system functions {pull}35946[#35946] (issue: {issue}35863[#35863])
+* Introduce INTERVAL support   {pull}35521[#35521] (issue: {issue}29990[#29990])
+
+Search::
+* Add intervals query {pull}36135[#36135] (issues: {issue}29636[#29636], {issue}32406[#32406])
+* Added soft limit to open scroll contexts #25244 {pull}36009[#36009] (issue: {issue}25244[#25244])
+
+[[enhancement-7.0.0-alpha2]]
 [float]
+=== Enhancements
+
+Aggregations::
+* Added keyed response to pipeline percentile aggregations 22302 {pull}36392[#36392] (issue: {issue}22302[#22302])
+* Enforce max_buckets limit only in the final reduction phase {pull}36152[#36152] (issues: {issue}32125[#32125], {issue}35921[#35921])
+* Histogram aggs: add empty buckets only in the final reduce step {pull}35921[#35921]
+* Handles exists query in composite aggs {pull}35758[#35758]
+* Added parent validation for auto date histogram {pull}35670[#35670]
+
+Analysis::
+* Allow word_delimiter_graph_filter to not adjust internal offsets {pull}36699[#36699] (issues: {issue}33710[#33710], {issue}34741[#34741])
+* Ensure TokenFilters only produce single tokens when parsing synonyms {pull}34331[#34331] (issue: {issue}34298[#34298])
+
+Audit::
+* Add "request.id" to file audit logs  {pull}35536[#35536]
+
+Authentication::
+* Invalidate Token API enhancements - HLRC {pull}36362[#36362] (issue: {issue}35388[#35388])
+* Add DEBUG/TRACE logs for LDAP bind {pull}36028[#36028]
+* Add Tests for findSamlRealm {pull}35905[#35905]
+* Add realm information for Authenticate API {pull}35648[#35648]
+* Formal support for "password_hash" in Put User {pull}35242[#35242] (issue: {issue}34729[#34729])
+
+Authorization::
+* Improve exact index matching performance {pull}36017[#36017]
+* `manage_token` privilege for `kibana_system` {pull}35751[#35751]
+* Grant .tasks access to kibana_system role {pull}35573[#35573]
+
+Build::
```

Comment:
We used to omit PRs with "Build" labels.  Should these Infra/Build PRs be omitted too?